src/org/labkey/test/components/ui/grids/ResponsiveGrid.java

```diff
@@ -184,11 +182,13 @@ protected void clickColumnMenuItem(String columnLabel, String menuText, boolean
 
         // scroll the header cell into view plus some extra vertical scroll to make sure the menu is visible
         getWrapper().scrollIntoView(headerCell);
+        getWrapper().shortWait().until(ExpectedConditions.elementToBeClickable(headerCell));    //
         getWrapper().scrollBy(0, 250);
```

Comment:
I was able to repro this locally. I don't know that the problem is with `ResponsiveGrid`. The behavior is just odd.
Scrolling the header into view works (`scrollIntoView(headerCell)`) but the extra scroll does not (`scrollBy(0, 250)`). It doesn't quite no-op though, it makes the subnav bar pop into view and calling `scrollBy` a second time actually scrolls down far enough to make the pop-up menu visible.
Adding a little extra scroll seems to do the trick.
```suggestion
        getWrapper().scrollIntoView(headerCell);
        getWrapper().scrollBy(0, 1); // First scroll fails sometimes
        getWrapper().scrollBy(0, 250);
```

Additionally, I wouldn't expect waiting for `elementToBeClickable` to do much more than add a slight delay here. "clickable" as far as that method is concerned mean enabled (`WebElement.isEnabled`) and not hidden(`WebElement.isDisplayed`). It doesn't know anything about whether an element is scrolled into view or blocked by another element.